src/Microsoft.IdentityModel.Tokens/Validation/Validators.Audience.cs

```diff
@@ -43,56 +43,40 @@ public static partial class Validators
         /// <exception cref="SecurityTokenInvalidAudienceException">If none of the 'audiences' matched either <see cref="TokenValidationParameters.ValidAudience"/> or one of <see cref="TokenValidationParameters.ValidAudiences"/>.</exception>
```

Comment:
Will you be updating the comments on this PR?

Reply:
No. We are still talking about further changes to the return type.
I believe we should make it our final step once everything is set in stone to have a pass through all the documentation for the new path and update / align it.